automated/linux/pibridge/test-pb-1.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+LOW=0
+HIGH=1
+
+#Configuration left side
+INPUT_L_1="I_1"
+INPUT_L_2="I_3"
+OUTPUT_L_1="O_1"
+OUTPUT_L_2="O_3"
+
+#Configuration right side
+INPUT_R_1="I_1_i03"
+INPUT_R_2="I_3_i03"
+OUTPUT_R_1="O_1_i05"
+OUTPUT_R_2="O_3_i05"
+
+set_piTest()
```

Comment:
Rename to something like the following
```suggestion
piTest_setOutput()
```
or better
```suggestion
pTest_setIOValue()
```